static_inverse_kinematics.py

```diff
@@ -39,6 +82,12 @@ def __init__(
     def _marker_diff(self, q: np.ndarray, xp_markers: np.ndarray):
         """
         Compute the difference between the marker position in the model and the position in the data
+        Arguments:
```

Comment:
add a space and replace "Arguments" by "Parameters"

Reply:
done